src/index.js

```diff
@@ -58,16 +59,30 @@ const unvisit = (input, schema, getDenormalizedEntity) => {
     return method(schema, input, unvisit, getDenormalizedEntity);
   }
 
+  if (input === undefined || input === null) {
```

Comment:
Wanted to call your attention to this change added in the last commit. This came up in the test with immutable records because it always instantiates all fields defined in the schema, using the default value if you didn't give it pass it one. E.g.

```js
> Menu = new Immutable.Record({ id: null, food: null })
[Function: Record]
> menuData = { id: 2 }
{ id: 2 }
> nonRecordMenu = menuData
{ id: 2 }
> recordMenu = new Menu(menuData)
Record { "id": 2, "food": null }
```

So while denormalizing the record, it's going to try to unvisit `food: null` which currently throws an unexpected error in `_getId`

This assumes that you're not going to have `undefined` or `null` resolve to anything during the denormalization process. Is that an ok assumption to make?